addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/discovery/AVMFritzDiscoveryService.java

```diff
@@ -104,14 +105,15 @@ protected synchronized void stopScan() {
      * @param device Device model received from a FRITZ!Box
      */
     public void onDeviceAddedInternal(DeviceModel device) {
-        ThingUID thingUID = this.bridgeHandler.getThingUID(device);
+        final ThingUID thingUID = bridgeHandler.getThingUID(device);
```

Comment:
Is there a specific reason to start doing it, I personally would be unhappy if the complete codebase would slowly be converted to this. @kaikreuzer wdyt?